lib/kitchen/directions/eoc_composite_page_container/v1.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::EocCompositePageContainer
+  class V1
+    renderable
+
+    def bake(metadata_source:, klass:, content:, append_to:, uuid_prefix: '.')
```

Comment:
`klass` here is not used as a CSS class -- it is used in an I18n key, as part of another klass, and in the UUID key.  That all of these uses need the same value is more of a coincidence and not a reason to actually make them the same variable.  So I think we should give it a different name, and I realize this means splitting it off from the `section_selector` variable up above.  I think splitting this one variable (that has many uses) into more than one variable is a good thing.

```ruby
def bake(metadata_source:, title_key:, uuid_key:, container_class_type:, content:, append_to:)
```

Note that by specifying the `uuid_key` you can get rid of the `uuid_prefix`.  I realize this makes the calls a little repetitive, but fortunately they are hidden inside other higher-level directions like `BakeChapterSummary` and won't be seen in the recipes themselves, generally.  Splitting out these variables will make the documentation and the readability better.

Reply:
👀 for clarification -- an example of this being called on, say, Further Research would be:

`bake(metadata_source: a MetadataElement, title_key: 'further-research', uuid_key: 'further-research'/'.further-research', container_class_type: 'section'/'div'/etc, content: '...', append_to: KitchenElement)`

I think a better name for `container_class_type` would be `section_element_name`, if I'm interpreting the purpose of that field correctly.

Reply:
There's a funky `/` in your example?

`container_class_type` is what gets interpolated into `os-blah-container`, so I don't know if `section_element_name` is right?  An element name is something like `h3`.

Reply:
oh the slashes are pseudocode

Reply:
I believe the above isn't a full set of the information this method would need. For Further Research, an example would be:

- title_key: 'further-research'
- uuid_key: either 'further-research' or '.further-research'
- container_class_type: 'os-further-research-container'
- section_selector: 'section.further-research' or 'div.further-research'
- metadata_source
- content
- append_to

Just to double-check, are you advocating that the first 4 items in that list should all be separate fields, despite the repetition?

Reply:
OOPS WAIT. I see that we don't need `section_selector` in `EocCompositePageContainer`. It would be the newly-renamed `MoveCustomSectionToEocContainer` that would need all of the first 4 fields passed in. Again, to double-check, is that your preferred implementation?